descriptors/ModuleDescriptor-template.json

```diff
@@ -190,6 +190,27 @@
         }
       ]
     },
+    {
+      "id": "voucher-storage.batch-vouchers",
+      "version": "2.0",
+      "handlers": [
+        {
+          "methods": ["POST"],
+          "pathPattern": "/batch-voucher-storage/batch-vouchers",
+          "permissionsRequired": ["voucher-storage.batch-vouchers.item.post"]
+        },
+        {
+          "methods": ["GET"],
+          "pathPattern": "/batch-voucher-storage/batch-vouchers/{id}",
+          "permissionsRequired": ["voucher-storage.batch-vouchers.item.get"]
+        },
+        {
+          "methods": ["DELETE"],
+          "pathPattern": "/batch-voucher-storage/batch-vouchers/{id}",
+          "permissionsRequired": ["voucher-storage.batch-vouchers.item.delete"]
```

Comment:
Please, rename permission to batch-voucher-storage.batch-vouchers.item.delete to correspond interface name.